src/Java.Interop/Java.Interop/JniTypeManagerOptions.cs

```diff
@@ -0,0 +1,10 @@
+namespace Java.Interop
+{
+	// These settings affect all instances of JniTypeManager
+	public static class JniTypeManagerOptions
+	{
+		// If `false`, reflection checks for the presence of the `JniAddNativeMethodRegistrationAttribute` attribute
+		// throughout the **entire** application will be skipped.
+		public static bool JniAddNativeMethodRegistrationAttributePresent { get; set; } = true;
```

Comment:
This need not exist here.  Instead, it should be in [`JniRuntime.CreationOptions`](https://github.com/xamarin/java.interop/blob/4f47ec82c14ae6ca23c1dcde3f7b5c8b93b6a4f7/src/Java.Interop/Java.Interop/JniRuntime.cs#L47-L71) and in `JniRuntime`.